contracts/MirrorNode.sol

```diff
@@ -28,6 +28,37 @@ abstract contract MirrorNode {
 
     function fetchTokenRelationshipOfAccount(string memory account, address token) external virtual returns (string memory json);
 
+    function fetchNonFungibleToken(address token, uint32 serial) external virtual returns (string memory json);
+
+    function getNftMetadata(address token, uint32 serial) external returns (string memory) {
+        try this.fetchNonFungibleToken(token, serial) returns (string memory json) {
```

Comment:
No need to wrap this in `try/catch`. If there is an error while fetching, I believe it's better to bubble the error up to the user.

Also, if the user has `ffi` disabled, this will mask the error. So better to remove the `try/catch`.